bots/cli/src/main/java/org/openjdk/skara/bots/cli/BotLauncher.java

```diff
@@ -163,20 +163,21 @@ public static void main(String... args) {
 
         applyLogging(jsonConfig);
         var log = Logger.getLogger("org.openjdk.skara.bots.cli");
+        log.info("Starting BotLauncher");
 
         BotRunnerConfiguration runnerConfig = null;
         try {
             runnerConfig = BotRunnerConfiguration.parse(jsonConfig, jsonFile.getParent());
         } catch (ConfigurationError configurationError) {
-            System.out.println("Failed to parse configuration file: " + jsonFile);
-            System.out.println("Error message: " + configurationError.getMessage());
+            log.severe("Failed to parse configuration file: " + jsonFile
```

Comment:
Will this actually log anything if there is no logging configured? `applyLogging` resets the `LogManager` and the `BotConsoleHandler` is only added if `log.console` is configured. Would it be wise to both print to standard out and log?